.gitignore

```diff
@@ -65,6 +65,9 @@ python-env/
 .idea
 
 Carthage/
+# Required for Glean
+/Generated
+.venv/
```

Comment:
need to update fx-ios readme to indicate the required python major version (i think virtual env will handle the minor version). prior to this PR only the build machines needed python.